README.md

```diff
@@ -1,42 +1,69 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/observIQ/stanza)](https://goreportcard.com/report/github.com/observIQ/stanza)
 [![License](https://github.com/observIQ/stanza/workflows/license/badge.svg)](https://github.com/observIQ/stanza/license)
 [![Gosec](https://github.com/observIQ/stanza/actions/workflows/gosec.yml/badge.svg)](https://github.com/observIQ/stanza/actions/workflows/gosec.yml)
 
+</center>
+
+# About Stanza
 
-<p align="center"><img src="https://github.com/observIQ/stanza/blob/master/docs/images/logo_small.png?raw=true"></p>
+Stanza is a fast and lightweight log transport and processing agent. It's designed as a modern replacement for Fluentd, Fluent Bit, and Logstash. Stanza can run as a standalone agent on all major operating systems, or it can be embedded directly into any Golang application. Stanza is also highly integrated to perform seamlessly with the applications in Google Cloud Platform (GCP) based production environments.
 
-# Stanza is moving to OpenTelemetry
+## OpenTelemetry
 
 Stanza has been contributed to the [OpenTelemetry](https://opentelemetry.io/) project and will be intergrated into the [OpenTelemetry collector](https://github.com/open-telemetry/opentelemetry-collector). 
-  
 
-# About Stanza
+# Features
 
-Stanza is a fast and lightweight log transport and processing agent. It's designed as a modern replacement for Fluentd, Fluent Bit, and Logstash. Stanza can run as a standalone agent on all major operating systems, or it can be embedded directly into any Golang application.
+- Flexible
+    - Supports many different input types such as file, journald, windows events, tcp / udp, and external APIs (cloudwatch, azure log analytics) as well as parsing with json and regex.
+    - Easily extended by writing an "operator" or "plugin" which is just a unit of code that performs a task such as reading data from a source, parsing data, or shipping data.
+- Lightweight with low resource consumption
+    - Uses next to no resource while idling. It does not pollute the system with tons of clutter, it exists strictly in /opt/observiq/stanza with just a few files.
+- Written in Go
+    - Everything is self contained into a single binary / file. There are no external dependency libraries that need to exist on the system, unlike dynamically linked applications (C, C++, C#, Java, Python, Ruby, NodeJS)
+- High Performance
+    - Stanza is proven to be significantly faster than fluentd and more stable at high throughput than fluentbit.
 
+## Supported Plugins
 
-# Quick Start
+Utilize Plugins to get up and running quickly. These are some of the different plugins supported by Stanza, with more being developed all the time.
 
-Want to get started right away? Check out our [Quick Start Guide](./docs/README.md)!
+<p align="center"><img src="docs/images/stanza_plugins.png?raw=true"></p>
 
+# Documentation
 
-## Community
+## [Quick Start](./docs/README.md)
```

Comment:
Should docs/README.md exist if it is basically part of the root readme? 

We might have external documentation / blogs linking to docs/README.md, so I would be hesitent to remove the file. Perhaps it could be updated saying something "moved to stanza/README.md"?

Reply:
docs/README.md is the quickstart guide whereas the main readme is more of an intro/marketing page